module/zfs/mmp.c

```diff
@@ -206,79 +206,61 @@ typedef enum mmp_vdev_state_flag {
 } mmp_vdev_state_flag_t;
 
 static vdev_t *
-mmp_random_leaf_impl(vdev_t *vd, int *fail_mask)
+get_head_and_rotate(list_t *list)
```

Comment:
nit: I'm not convinced we want a separate function for this, my feeling is it doesn't improve the readability any.

Reply:
Done